### PR TITLE
Translate my desc

### PR DESCRIPTION
**Pull request by chybie**

### CODE REVIEW

File: site/_data/authorsData.json [hunk 1]
```diff
@@ -334,8 +334,11 @@
     "descriptions": {
       "en": "Developer advocate working on Chrome DevTools at Google.",
       "es": "Developer advocate, trabajando en Chrome DevTools en Google.",
-      "zh": "现为谷歌 Chrome DevTools 的开发者推广工程师。",
-      "ru": "Developer advocate, rabotayushchiy nad Chrome DevTools v Google."
+      "zh": "现为谷歌 Chrome DevTools 的开发技术推广工程师。",
+      "ru": "Developer advocate, rabotayushchiy nad Chrome DevTools v Google.",
+      "pt": "Defensor do desenvolvedor trabalhando no Chrome DevTools no Google.",
+      "ko": "Google의 Chrome DevTools에서 일하는 개발자 옹호자입니다.",
+      "ja": "開発者は、GoogleでChromeDevToolsに取り組んでいます。"
```
Comment (chybie) [inline site/_data/authorsData.json:341]: @agektmr is this translation correct? (I Google translate it)

File: site/_data/authorsData.json [hunk 2]
```diff
@@ -334,8 +334,11 @@
     "descriptions": {
       "en": "Developer advocate working on Chrome DevTools at Google.",
       "es": "Developer advocate, trabajando en Chrome DevTools en Google.",
-      "zh": "现为谷歌 Chrome DevTools 的开发者推广工程师。",
-      "ru": "Developer advocate, rabotayushchiy nad Chrome DevTools v Google."
+      "zh": "现为谷歌 Chrome DevTools 的开发技术推广工程师。",
+      "ru": "Developer advocate, rabotayushchiy nad Chrome DevTools v Google.",
+      "pt": "Defensor do desenvolvedor trabalhando no Chrome DevTools no Google.",
```
Comment (chybie) [inline site/_data/authorsData.json:339]: @andreban is this translation correct? (I Google translate it)

File: site/_data/authorsData.json [hunk 3]
```diff
@@ -334,8 +334,11 @@
     "descriptions": {
       "en": "Developer advocate working on Chrome DevTools at Google.",
       "es": "Developer advocate, trabajando en Chrome DevTools en Google.",
-      "zh": "现为谷歌 Chrome DevTools 的开发者推广工程师。",
-      "ru": "Developer advocate, rabotayushchiy nad Chrome DevTools v Google."
```
Comment (chybie) [inline site/_data/authorsData.json:338]: @kateryna-prokopenko  is this translation correct? (I Google translate it)

File: site/_data/authorsData.json [hunk 4]
```diff
@@ -334,8 +334,11 @@
     "descriptions": {
       "en": "Developer advocate working on Chrome DevTools at Google.",
       "es": "Developer advocate, trabajando en Chrome DevTools en Google.",
```
Comment (chybie) [inline site/_data/authorsData.json:336]: @and-oli  is this translation correct? (I Google translate it)

File: site/_data/authorsData.json (hunk 1 above)
Comment (agektmr) [reply site/_data/authorsData.json:341]:
```suggestion
      "ja": "Google で Chrome DevTools に取り組んでいる Developer Advocate です。"
```
I'd fix it to this.

File: site/_data/authorsData.json (hunk 2 above)
Comment (andreban) [reply site/_data/authorsData.json:339]: I wouldn't translate "Developer Advocate", and just use the english term in this case.

```suggestion
      "pt": "Developer Advocate trabalhando no Chrome DevTools no Google.",
```

**Review by samthor — APPROVED** [review]

**Pull request merged**